### PR TITLE
Wider main content container in the documentation.

### PR DESCRIPTION
**Pull request by hvy**
## Motivation

Rightmost space in the documentation is unused (for desktop). Content container can be wider aiding readability and reducing content height.

## Description of the changes

Increases the width of container of the main content in the docs. Tables for instance are more compact.

### CODE REVIEW

**Comment by HideakiImamura** [comment]
Thanks for the PR. In the current document for desktop, the navigation bar at the top of pages is in the middle of the page. I think it is better to move the navigation bar to align a wider content container.

スクリーンショット 2020-07-06 17 06 15

**Comment by hvy** [comment]
Makes sense, fixes. Thanks for pointing that out. PTAL.

File: docs/source/_static/css/custom.css [hunk 1]
```diff
@@ -41,6 +41,11 @@ em.sig-param > span:not(:nth-child(1)):not(:nth-child(2)) {
   border-left: solid 3px #6ab0de;
 }
 
+/* Main content */
+.wy-nav-content {
+    max-width: 1200px;
```
Comment (toshihikoyanase) [inline docs/source/_static/css/custom.css:46]: A nit, but could you use two whitespaces for indent? The current code format is introduced by #1463 using https://prettier.io, and I think we need to discuss the formatting rule for HTML/CSS if we change it.
```suggestion
  max-width: 1200px;
```

Comment (hvy) [reply docs/source/_static/css/custom.css]: Ops, should've noticed that. Thanks, applied.

**Comment by hvy** [comment]
PTAL.

**Pull request merged**